pkg/controller/healthz.go

```diff
@@ -16,15 +18,21 @@ import (
 
 var (
 	rl = rate.NewLimiter(rate.Every(time.Minute), 1)
+
+	mHealthzBuildInfo = stats.Int64("build_info", "Current build information", "1")
 )
 
-func HandleHealthz(hctx context.Context, cfg *database.Config, h *render.Renderer) http.Handler {
-	logger := logging.FromContext(hctx).Named("healthz")
+func HandleHealthz(ctx context.Context, cfg *database.Config, h *render.Renderer) http.Handler {
+	logger := logging.FromContext(ctx).Named("healthz")
```

Comment:
Why not create the view here, this function is only called once right?

Reply:
It's called multiple times in the e2e/performance things.